### ISSUE TITLE
IBM JDK Support

### ISSUE THREAD

**Issue by jamesnetherton** [OpenHFT/Chronicle-Core#44]
Are there any plans for this project to support the IBM JDK?

I've been testing the Apache Camel [chronicle-engine component](https://github.com/apache/camel/blob/master/components/camel-chronicle/src/main/docs/chronicle-engine-component.adoc) which uses various OpenHFT libraries. On IBM JDK I hit:

```
Caused by: java.lang.ClassNotFoundException java.lang.AbstractStringBuilder
	at net.openhft.chronicle.core.util.StringUtils.<clinit>(StringUtils.java:50)
	at net.openhft.chronicle.core.pool.StringBuilderPool.acquireStringBuilder(StringBuilderPool.java:32)
	at net.openhft.chronicle.wire.Wires.acquireStringBuilder(Wires.java:149)
	at net.openhft.chronicle.engine.map.remote.RemoteKVSSubscription.toUri(RemoteKVSSubscription.java:65)
	at net.openhft.chronicle.engine.map.remote.RemoteKVSSubscription.<init>(RemoteKVSSubscription.java:57)
	at net.openhft.chronicle.engine.tree.VanillaAsset$$Lambda$1385.0000000025193610.create(Unknown Source)
	at net.openhft.chronicle.engine.tree.VanillaAsset.createLeafView(VanillaAsset.java:350)
	at net.openhft.chronicle.engine.tree.VanillaAsset.createLeafView(VanillaAsset.java:353)
	at net.openhft.chronicle.engine.tree.VanillaAsset.createLeafView(VanillaAsset.java:353)
	at net.openhft.chronicle.engine.tree.VanillaAsset.lambda$acquireView$8(VanillaAsset.java:413)
	at net.openhft.chronicle.engine.tree.VanillaAsset$$Lambda$1395.0000000026B90ED0.call(Unknown Source)
	at net.openhft.chronicle.threads.Threads.withThreadGroup(Threads.java:42)
	at net.openhft.chronicle.engine.tree.VanillaAsset.acquireView(VanillaAsset.java:412)
	at net.openhft.chronicle.engine.api.tree.AssetTree.acquireSubscription(AssetTree.java:283)
	at net.openhft.chronicle.engine.api.tree.AssetTree.registerSubscriber(AssetTree.java:208)
	at org.apache.camel.component.chronicle.engine.ChronicleEngineConsumer.doStart(ChronicleEngineConsumer.java:58)
```

**Comment by peter-lawrey** [comment]
I will add it to the roadmap, however I don't think we will have this
working before Q1 next year as none of our clients use the IBM JVM.
ᐧ

**Comment by RobAustin** [comment]
@jamesnetherton if you are interested for "IBM JDK Support #44" please contact sales@chroncile.software

**Comment by hft-team-city** [comment]
Released in [Chronicle-Core-2.20.102](https://github.com/OpenHFT/Chronicle-Core/releases/chronicle-core-2.20.102), [BOM-2.20.134](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.134)
